platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDbCompactionCoordinator.java

```diff
@@ -50,10 +51,14 @@
  * The number of threads in the pool is defined by {@link MerkleDbConfig#compactionThreads()} property.
  *
  */
+@SuppressWarnings("rawtypes")
 class MerkleDbCompactionCoordinator {
 
     private static final Logger logger = LogManager.getLogger(MerkleDbCompactionCoordinator.class);
 
+    // Timeout to wait for all currently running compaction tasks to stop during compactor shutdown
+    private static final long SHUTDOWN_TIMEOUT = 60_000;
```

Comment:
It's best to include the units of the value in the name of the constant, e.g. by adding a `_MILLIS` suffix.

Reply:
OK